WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -760,6 +760,7 @@ private void showQuickStartDialog() {
                               );
 
         promoDialog.show(getSupportFragmentManager(), tag);
+        AnalyticsTracker.track(AnalyticsTracker.Stat.QUICK_START_REQUEST_SEEN);
```

Comment:
We can remove the unnecessary `AnalyticsTracker` qualification from `AnalyticsTracker.Stat.QUICK_START_REQUEST_SEEN` to produce `AnalyticsTracker.track(Stat.QUICK_START_REQUEST_SEEN);` for simplicity.